packages/fxa-settings/src/styles/fonts.scss

```diff
@@ -0,0 +1,44 @@
+@font-face {
+  font-family: 'Inter';
+  font-style:  normal;
+  font-weight: 400;
+  font-display: swap;
```

Comment:
I think `swap` is a good choice for now, if we notice any delayed loading and don't like seeing a fallback switch to the actual font we can try `block` or just resort to `fallback`.